src/lib/Scenes/Fair/Components/FairHeader/index.tsx

```diff
@@ -49,10 +49,74 @@ const CountdownContainer = styled.View`
   width: 100%;
 `
 
+const Bold = styled(Sans)`
+  font-weight: bold;
+`
 export class FairHeader extends React.Component<Props> {
+  getArtists(artistCounts, artistsNames) {
+    let { artists } = artistCounts
+    let firstArtist
+    let lastArtist
+
+    if (artistsNames.edges[0] && artistsNames.edges[0].node && artistsNames.edges[0].node.name) {
+      firstArtist = artistsNames.edges[0].node.name + ", "
+      artists = artists - 1
+    }
+    if (artistsNames.edges[1] && artistsNames.edges[1].node && artistsNames.edges[1].node.name) {
+      lastArtist = artistsNames.edges[1].node.name + ", "
+      artists = artists - 1
+    }
+
+    return (
+      <Flex flexDirection="row" flexWrap="wrap">
+        <Sans size="3">Works by </Sans>
+        <Bold size="3">{firstArtist}</Bold>
+        <Bold size="3">{lastArtist}</Bold>
+        <Sans size="3">{"and "}</Sans>
+        <Bold size="3">{artists + " others."}</Bold>
+      </Flex>
+    )
+  }
+  getExhibitors(exhibitors) {
+    let exhibitorCount = 0
+    const exhibitorNamesToDisplay = []
+    const areOtherExhibitorsToDisplay = exhibitors.length > 4 ? true : false
+
+    exhibitors.forEach(group => {
+      if (exhibitorNamesToDisplay.length < 4) {
+        exhibitorNamesToDisplay.push(first(group.exhibitors))
+      }
+      exhibitorCount += group.exhibitors.length
+    })
+
+    if (areOtherExhibitorsToDisplay) {
+      return (
+        <Flex flexDirection="row" flexWrap="wrap">
+          <Sans size="3">{"From "}</Sans>
+          <Bold size="3">{exhibitorNamesToDisplay[0] + ", "}</Bold>
+          <Bold size="3">{exhibitorNamesToDisplay[1] + ", "}</Bold>
+          <Sans size="3">{"and "}</Sans>
+          <Bold size="3">{exhibitorCount - 2 + " others"}</Bold>
+        </Flex>
+      )
+    }
+
+    // Handle instance when there are 3 or fewer exhibitors to display
+    const shortExhibitorList = [<Sans size="3">{"From "}</Sans>]
+
+    exhibitorNamesToDisplay.forEach((exhibitor, index) => {
+      if (index === exhibitorNamesToDisplay.length - 1) {
+        shortExhibitorList.push(<Sans size="3">{" and "}</Sans>, <Bold size="3">{exhibitor}</Bold>)
+      } else {
+        shortExhibitorList.push(<Bold size="3">{exhibitor + ", "}</Bold>)
+      }
+    })
+
+    return shortExhibitorList
```

Comment:
Sorry for missing this first time around but you could write this method more succinctly by handling a "short" list of exhibitors at the same time as the long one.

Taking this one step further you could actually abstract all that logic in one component for both the list of artists and the list of exhibitors as they're essentially the same like I did in the `ArtistNamesList` for the Show header view
https://github.com/artsy/emission/blob/master/src/lib/Scenes/Show/Components/ShowHeader/Components/ArtistNamesList.tsx